doc/CustomRules.md

```diff
@@ -51,7 +51,7 @@ A rule is implemented as an `Object` with one optional and four required propert
     - `config` is an `Object` corresponding to the rule's entry in `options.config` (if present).
   - `onError` is a function that takes a single `Object` parameter with one required and four optional properties:
     - `lineNumber` is a required `Number` specifying the 1-based line number of the error.
-    - `details` is an optional `String` with information about what caused the error.
+    - `detail` is an optional `String` with information about what caused the error.
```

Comment:
There is an "s" https://github.com/DavidAnson/markdownlint/blob/f3f8d635d58387cc78d0a66e394a0ad925b3301f/lib/markdownlint.d.ts#L211

Reply:
The types and documentation say `details`, but internally `detail` is used.